array_analyzer/workflows/registration_workflow.py

```diff
@@ -110,21 +113,24 @@ def point_registration(input_dir, output_dir):
             spot_coords=spot_coords,
             particles=particles,
             stds=np.array(constants.STDS),
-            debug=constants.DEBUG,
+            logger=logger,
         )
-        if min_dist / nbr_fiducials > constants.REG_DIST_THRESH:
-            warnings.warn("Registration failed, repeat with outlier removal")
+        reg_dist = min_dist / nbr_fiducials
+        if reg_dist > constants.REG_DIST_THRESH:
+            logger.warning("Registration failed, repeat with outlier removal")
```

Comment:
This is a minor point but it could be useful to log the well where the initial registration fails as well.